hieradata/common.yaml

```diff
@@ -610,6 +610,8 @@ govuk_gor::apt_mirror_hostname: "%{hiera('apt_mirror_hostname')}"
 
 govuk_htpasswd::http_username: "%{hiera('http_username')}"
 
+govuk_jenkins::jobs::deploy_app::graphite_host: 'graphite.cluster'
```

Comment:
If this is the default we do not need to also have it in hieradata.

Reply:
That's true. Removed.